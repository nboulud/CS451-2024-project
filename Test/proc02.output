b 8
b 10
b 2
b 3
b 7
b 9
b 6
b 5
b 1
b 4
