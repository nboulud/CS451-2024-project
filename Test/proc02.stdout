passe par main
Message envoyé par 2contenant  1
Message envoyé par 2contenant  2
Message envoyé par 2contenant  3
Message envoyé par 2contenant  4
Message envoyé par 2contenant  5
Message envoyé par 2contenant  6
Message envoyé par 2contenant  7
Message envoyé par 2contenant  8
Message envoyé par 2contenant  9
Message envoyé par 2contenant  10
Message envoyé par 2contenant  1
Message envoyé par 2contenant  2
Message envoyé par 2contenant  3
Message envoyé par 2contenant  4
Message envoyé par 2contenant  5
Message envoyé par 2contenant  6
Message envoyé par 2contenant  7
Message envoyé par 2contenant  8
Message envoyé par 2contenant  9
Message envoyé par 2contenant  10
Message envoyé par 2contenant  1
Message envoyé par 2contenant  2
Message envoyé par 2contenant  3
Message envoyé par 2contenant  4
Message envoyé par 2contenant  5
Message envoyé par 2contenant  6
Message envoyé par 2contenant  7
Message envoyé par 2contenant  8
Message envoyé par 2contenant  9
Message envoyé par 2contenant  10
Message reçu de receiver contenant ack 8
Message reçu de receiver contenant ack 10
Message reçu de receiver contenant ack 2
Message reçu de receiver contenant ack 3
Message reçu de receiver contenant ack 7
Message envoyé par 2contenant  1
Message envoyé par 2contenant  4
Message envoyé par 2contenant  5
Message envoyé par 2contenant  6
Message envoyé par 2contenant  9
Message reçu de receiver contenant ack 9
Message reçu de receiver contenant ack 6
Message reçu de receiver contenant ack 5
Message reçu de receiver contenant ack 1
Message envoyé par 2contenant  4
Message envoyé par 2contenant  4
Message reçu de receiver contenant ack 4
All messages acknowledged. Sender terminating.
My PID: 16081

From a new terminal type `kill -SIGINT 16081` or `kill -SIGTERM 16081` to stop processing packets

My ID: 2

List of resolved hosts is:
==========================
1
Human-readable IP: 127.0.0.1
Human-readable Port: 11001

2
Human-readable IP: 127.0.0.1
Human-readable Port: 11002

3
Human-readable IP: 127.0.0.1
Human-readable Port: 11003


Path to output:
===============
/home/dcl/Project/DA/gitda/Test/proc02.output

Path to config:
===============
./../Test/config

Doing some initialization

Broadcasting and delivering messages...

Immediately stopping network packet processing.
Writing output.
